fusor-ember-cli/app/routes/review/progress/overview.js

```diff
@@ -3,12 +3,11 @@ import Ember from 'ember';
 export default Ember.Route.extend({
   model: function () {
       var deployment = this.modelFor('deployment');
-      var deployTaskPromise = this.store.find('foreman-task', deployment.get('foreman_task_uuid'));
+      var deployTaskPromise = this.store.find('foreman-task', {search: "id = " + deployment.get('foreman_task_uuid')});
       var subtasksOfDeployPromise = this.store.find('foreman-task', {search: "parent_task_id = " + deployment.get('foreman_task_uuid')});
       var self = this;
-
       return Ember.RSVP.Promise.all([deployTaskPromise, subtasksOfDeployPromise]).then(function(results) {
-        var deployTask = results[0];
+        var deployTask = results[0].get('firstObject');
```

Comment:
So, the return object iof the scoped_search query s now an array, and it will always have only have one object, so we take the first one.